project.clj

```diff
@@ -119,6 +120,7 @@
                                            [day8.re-frame/tracing "0.5.3"]
                                            [ring/ring-devel "1.8.0"]
                                            [ring/ring-mock "0.4.0"]
+                                           [ring/ring-json "0.5.0"]
```

Comment:
i don't think this dependency is necessary, as ring.json is already in the master list above

Reply:
Done